client/src/components/App.jsx

```diff
@@ -1,31 +1,45 @@
 import React from 'React';
-import ReactDOM from 'react-dom';
-import $ from 'jquery';
-
+import axios from 'axios';
+import Menu from './Menu.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      showMenu: false,
+    }
     this.getData = this.getData.bind(this);
+    this.showMenu = this.showMenu.bind(this);
```

Comment:
For methods that you are not passing down to sub-components I don't think you will need to bind them. I see that you use hideMenu in the menu component so that will require a binding but the other two look like they do not as of right now.

Reply:
Other than that really nice work here

Reply:
Oh yeah I was getting some undefined errors when trying to call setState with the showMenu method within the render so binding it ended up working, I'm not sure if this is best practice though